src/remote/document.js

```diff
@@ -0,0 +1,21 @@
+/* @flow */
+
+// The remote Cozy metadata, as returned by cozy-client-js
+export type RemoteDoc = {
+  _id: string,
+  _rev: string,
+  _type: string,
+  class?: string,
+  created_at: string,
+  dir_id: string,
+  executable?: boolean,
+  md5sum?: string,
+  mime?: string,
+  name: string,
+  path?: string,
+  size?: string,
+  tags: string[],
+  type: string,
```

Comment:
`_type` is the doctype, while type is the "files" API attribute.

I was a bit confused about `cozy-client-js`'s `find()` method returning this, and the cozy-stack using a nested `attributes` object...